flytekit/common/launch_plan.py

```diff
@@ -88,6 +93,25 @@ def register(self, project, domain, name, version):
         self._id = id_to_register
         return str(self.id)
 
+    @_exception_scopes.system_entry_point
+    def fast_register(self, project, domain, name, already_uploaded_digest=None) -> str:
+        """
+        :param Text project: The project in which to register this task.
+        :param Text domain: The domain in which to register this task.
+        :param Text name: The name to give this task.
+        :param Text already_uploaded_digest: The version in which to register this task (if it's not already computed).
+        :rtype: Text: Registered identifier.
+        """
+        digest = already_uploaded_digest
+        if already_uploaded_digest is None:
+            cwd = _Path(_os.getcwd())
```

Comment:
so we will upload everything from cwd? it the assumption that user runs this at the root?
Can we make cwd as the default and ask the user for the root_path?

Reply:
done